autobuild/build_util.py

```diff
@@ -88,6 +76,30 @@ def execute_notebook(f):
         # subprocess.run(['jupyter', 'nbconvert', '--to', 'notebook', '--execute', f'{f}'], check=True)
 
 
+def exexcute_notebooks_in_folder(ROOT_PATH, no_run_list=None):
```

Comment:
I keep meaning to fix this spelling

Reply:
In this context we could use a default value of () for no_run_list as tuples are immutable